diff --git a/logging/env_logger.h b/logging/env_logger.h
index b9099a834..0aa2b768a 100644
--- a/logging/env_logger.h
+++ b/logging/env_logger.h
@@ -57,12 +57,12 @@ class EnvLogger : public Logger {
       // Preserve iostats not to pollute writes from user writes. We might
       // need a better solution than this.
       SetPerfLevel(PerfLevel::kDisable);
-      iostats_context.disable_iostats = true;
+      // iostats_context.disable_iostats = true;
       logger.mutex_.Lock();
     }
     ~FileOpGuard() {
       logger_.mutex_.Unlock();
-      iostats_context.disable_iostats = false;
+      // iostats_context.disable_iostats = false;
       SetPerfLevel(prev_perf_level_);
     }
 
